raiden-cli/src/routes/api.v1.ts

```diff
@@ -19,15 +19,24 @@ export function makeApiV1Router(this: Cli): Router {
   router.use('/testing', makeTestingRouter.call(this));
 
   router.get('/version', (_request: Request, response: Response) => {
+    response.json({ version: Raiden.version });
+  });
+
+  router.get('/contracts', (_request: Request, response: Response) => {
+    const contracts = this.raiden.contractsInfo;
     response.json({
-      version: Raiden.version,
+      contracts_version: Raiden.contractVersion,
```

Comment:
Similar to an earlier issue. We should maybe consider to remove old interface definitions for API objects. Or if you still see an advantage of them add one for this route endpoint as well.

Reply:
As I replied there, let's decide on this and do it on a separate PR.